src/Plugins/PHPBugs.php

```diff
@@ -89,7 +89,8 @@ private function checkBugs()
 
             foreach ($this->rooms as $room) {
                 yield $this->chatClient->postMessage($room, \sprintf(
-                    "[tag:php] [tag:bug] %s – [#%d](%s)",
+                    "[tag:php] [tag:%s] %s – [#%d](%s)",
+                    $bug["type"],
```

Comment:
What is the range of possible values for this?

Reply:
Bug, Doc, Req, and SecBug

Reply:
Could you map that to `bug`, `documentation`, `feature-request`, `security`?